dist/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+<script defer src="index.bundle.js"></script></head>
+<body>
+    <script src="main.js"></script>
```

Comment:
You can remove later the script src="main.js", as the webpack creates the index.bundle.js